app/Http/Client/Alchemy/AlchemyPendingRequest.php

```diff
@@ -337,6 +337,10 @@ public function parseNft(array $nft, int $networkId): Web3NftData
             $description = Arr::get($nft, 'description.0');
         }
 
+        if ($description === null) {
```

Comment:
Just to be safe we should always perform the above check on the description, since Alchemy can be very inconsistent:

```php
 if (is_array($description)) {
     $description = Arr::get($nft, 'description.0');
 }
```